config/config.json

```diff
@@ -0,0 +1,25 @@
+{
+  "monitoringCluster": "mad",
+  "logDirectory": "logs",
+  "pipelinesDirectory": "/opt/mad/config/pipelines",
+  "httpHost": "0.0.0.0",
+  "httpPort": 7090,
+  "httpHealthCheckPath": "/ping",
+  "httpStatusPath": "/status.json",
```

Comment:
Thoughts us just using /status? -> implication being for CAgg that we merge status.json and the akka status endpoint.